### DISCUSSION TITLE
Development branch vs. previous version

### DISCUSSION THREAD

**Discussion by laurajohanne** [aertslab/scenicplus#339] [Q&A]
Hi,

I have two questions, and I hope you can shed some light on them:

1. After running both the development branch and the previous version, I noticed a difference in the scenicplus object. In the previous version, it had the key eRegulon_AUC, whereas the development branch has eRegulon_region_AUC and eRegulon_gene_AUC. When plotting a UMAP based on this data, the previous version (with eRegulon_AUC) seemed to consist of both region and gene information, whereas the development branch appears to plot the information separately in the UMAP. I'm wondering if the development branch has or is perhaps getting a similar feature with combined data when plotting UMAP? I experienced higher resolution in my data in the UMAP when using the previous version, and I want to replicate that with the development branch.

2. After running the development branch, I identified a lot of eRegulons, so when plotting the heatmap_dotplot, it is difficult to interpret the figure. Is there a way to subset high-quality eRegulons in the development branch as there was in the previous version? Should I still use "rho" to filter the eRegulons, or should I use the new factor "triplet rank"?

Thank you in advance. Looking forward to your guidance on these matters

Best regards,
Laura

**Comment by SeppeDeWinter** [comment]
Hi Laura

Question 1:
Yes, this is still possible. You have to create an AnnData object that has both matrices concatenated and then generate a umap based on that using functions from scanpy.

Question 2:
You can use the correlation between region based AUC and gene based AUC (for this you can use the code in this file https://github.com/aertslab/scenicplus/blob/development/src/scenicplus/regulon_qc/quality_metrics.py). Or you can use the triplet rank. Both are valid options.

Let me know if you get stuck with any of this.

All the best,
Seppe

**Reply by laurajohanne** [reply]
Hi Seppe,

Thank you very much for reply. I will try it out. 

Best regards Laura
